tests/integration/models/setup_utils.py

```diff
@@ -22,47 +26,47 @@ class RectangularDomainThreeFractures(pp.ModelGeometry):
     params: dict
     """Parameters for the model."""
 
-    def set_fracture_network(self) -> None:
+    def set_fractures(self) -> None:
         # Length scale:
         ls = 1 / self.units.m
 
         fracture_indices = self.params.get("fracture_indices", [0])
-        domain = pp.Domain({"xmin": 0, "xmax": 2 * ls, "ymin": 0, "ymax": 1 * ls})
         fractures = [
             pp.LineFracture(np.array([[0, 2], [0.5, 0.5]]) * ls),
             pp.LineFracture(np.array([[0.5, 0.5], [0, 1]]) * ls),
             pp.LineFracture(np.array([[0.3, 0.7], [0.3, 0.7]]) * ls),
         ]
-        used_fractures = [fractures[i] for i in fracture_indices]
-        self.fracture_network = pp.FractureNetwork2d(used_fractures, domain)
+        self._fractures = [fractures[i] for i in fracture_indices]
 
-    def mesh_arguments(self) -> dict:
+    def meshing_arguments(self) -> dict:
         # Divide by length scale:
-        h = 0.5 / self.units.m
-        return {"mesh_size_frac": h, "mesh_size_bound": h}
+        ls = 1 / self.units.m
+        mesh_sizes = {
+            # Cartesian: 2 by 8 cells.
+            "cell_size_x": 0.25 * ls,
+            "cell_size_y": 0.5 * ls,
+            # Simplex. Whatever gmsh decides.
+            "cell_size_fracture": 0.5 * ls,
+            "cell_size_boundary": 0.5 * ls,
+            "cell_size_min": 0.2 * ls,
```

Comment:
As a matter of information, by setting `"cell_size_min": 0.5 * ls`, the cell size will be `0.5 * ls`.